### PR TITLE
test file link to image in readme

### PR DESCRIPTION
**Pull request by Jessicashinjo**

### CODE REVIEW

**Comment by carter-harris** [comment]
👍

**Comment by ashwoodaa** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -1,90 +1,49 @@
...
-> **Instructor guidance:** You may want to add more domains to the lexicon as you identify them.
+### Installation
+---
+If you're on Mac OS X, consider installing with [Homebrew](https://github.com/yyuu/pyenv#homebrew-on-mac-os-x). Run the command ```$ python --version``` from your command line , we recommend Python 3.3.6
 
-##### Example
+If you do not have pyenv use this [link](https://github.com/yyuu/pyenv#homebrew-on-mac-os-x) to install.
 
-> Friendship is like money, easier made than kept.
+This project was built using Python 3.3, click [here](https://www.python.org/download/releases/3.3.0/) to install this version.
```
Comment (carter-harris) [inline README.md:32]: right here we could add the following....

 $ git clone https://github.com/nss-day-cohort-13/language-analyzer-lazer-powered-cthulhu.git```

$ cd language-analyzer-lazer-powered-cthulhu```

**Comment by keithbradley1** [comment]
I like Carter's addition

**Comment by carter-harris** [comment]
👍

**Comment by ashwoodaa** [comment]
:+1:

**Comment by keithbradley1** [comment]
👍

**Pull request merged**